service/src/main/java/uk/gov/hmcts/reform/fpl/model/DocmosisProceeding.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class DocmosisProceeding {
+    private final String onGoingProceeding;
+    private final String proceedingStatus;
+    private final String caseNumber;
```

Comment:
what is caseNumber here?

Reply:
As part of Other Proceedings sections, we can provide caseNumber of the other proceedings